Use DTS to enable data and instruction prefetch.

NOTE: This also silences the kernel warning/error message about register corruption:
 | L2C OF: override cache size: 524288 bytes (512KB)
 | L2C OF: override line size: 32 bytes
 | L2C OF: override way size: 32768 bytes (32KB)
 | L2C OF: override associativity: 16
 | L2C: platform provided aux values permit register corruption.

NOTE2: Following warning is still thrown and can only be removed by disabling i/d prefetch (set prefetch-data and prefetch-instr to 0)
 | L2C: DT/platform modifies aux control register: 0x02050000 -> 0x32050000

diff -aurN a/arch/arm/boot/dts/netx4000/netx4000-common.dtsi b/arch/arm/boot/dts/netx4000/netx4000-common.dtsi
--- a/arch/arm/boot/dts/netx4000/netx4000-common.dtsi	2022-03-31 07:38:56.232759949 +0200
+++ b/arch/arm/boot/dts/netx4000/netx4000-common.dtsi	2022-04-25 10:05:15.364018068 +0200
@@ -78,9 +78,8 @@
 		reg = <0xfaf10000 0x1000>;
 		cache-unified;
 		cache-level = <2>;
-		cache-size = <524288>;
-		cache-sets = <1024>;
-		cache-line-size = <32>;
+		prefetch-data = <1>;
+		prefetch-instr = <1>;
 	};
 
 	/* Global Interrupt Controller */
diff -aurN a/arch/arm/mach-netx4000/board-generic.c b/arch/arm/mach-netx4000/board-generic.c
--- a/arch/arm/mach-netx4000/board-generic.c	2022-03-31 07:38:47.364553700 +0200
+++ b/arch/arm/mach-netx4000/board-generic.c	2022-04-25 08:40:13.300613865 +0200
@@ -94,7 +94,7 @@
 	.reserve      = NULL,
 	.smp          = smp_ops(netx4000_smp_ops),
 	.map_io       = netx4000_map_io,
-	.l2c_aux_val  = L310_AUX_CTRL_DATA_PREFETCH | L310_AUX_CTRL_INSTR_PREFETCH,
+	.l2c_aux_val  = 0,
 	.l2c_aux_mask = ~0,
 	.init_early   = NULL,
 	.init_irq     = NULL, /* not needed since it is part of DT -> gic */
